lints/base.go

```diff
@@ -207,6 +211,11 @@ type LintReport struct {
 	ERsaNoPublicKey                                      ResultStruct `json:"e_rsa_no_public_key,omitempty"`
 	ESubCertCertificatePoliciesMissing                   ResultStruct `json:"e_sub_cert_certificate_policies_missing,omitempty"`
 	ESubCertKeyUsageCrlSignBitSet                        ResultStruct `json:"e_sub_cert_key_usage_crl_sign_bit_set,omitempty"`
+	infosPresent                                         bool         `json:"infos_present,omitempty"`
```

Comment:
Why are there five vars here, but everywhere else in this PR, there are only four?